composer.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "yiisoft/yii-base-cli",
+    "type": "library",
+    "description": "Yii Framework Web Application Base",
+    "keywords": ["yii", "framework", "application"],
+    "homepage": "https://github.com/yiisoft/yii-base-web",
+    "license": "BSD-3-Clause",
+    "support": {
+        "source": "https://github.com/yiisoft/yii-base-cli",
+        "issues": "https://github.com/yiisoft/yii-base-cli/issues",
+        "forum": "http://www.yiiframework.com/forum/",
+        "wiki": "http://www.yiiframework.com/wiki/",
+        "irc": "irc://irc.freenode.net/yii"
+    },
+    "authors": [
```

Comment:
We can remove it, I think. Makes little sense nowadays with such many contributors.